CHANGELOG.md

```diff
@@ -76,6 +76,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Do not use spaces around variable assignment in generated embed framework script
```

Comment:
nit add two empty spaces at the end of this line for formatting.

Reply:
also move entry above under `master` section, I believe this is already 1.7.0.beta.1